src/plugins/external_tool_support/src/utils/ExternalToolSearchTask.cpp

```diff
@@ -53,6 +59,13 @@ void ExternalToolSearchTask::run() {
 
     if (toolsDir.isEmpty() && QFileInfo(appDir.absoluteFilePath(DEFAULT_TOOLS_DIR_NAME)).isDir()) {
         toolsDir = appDir.absoluteFilePath(DEFAULT_TOOLS_DIR_NAME);
+#ifdef Q_OS_DARWIN
+    } else if (toolsDir.isEmpty()) {
+        QString dir = BundleInfoMac::getToolsSearchPath();
+        if (QFileInfo(dir).isDir()) {
```

Comment:
You should compare with an empty string here: the getToolsSearchPath already checks if the dir exists.

Reply:
done